src/base_model.jl

```diff
@@ -675,6 +723,7 @@ function calculate_self_consumption(ECModel::AbstractEC; per_unit::Bool=true)
     shared_cons_us = JuMP.Containers.DenseAxisArray(
         Float64[sum(time_res .* energy_weight .* max.(0.0, 
                 sum(profile_component(users_data[u], l, "load") for l in asset_names(users_data[u], LOAD)) 
+                .+ sum(ECModel.results[:P_adj_tot_us][u, :])
                 + min.(_P_us[u, :], 0.0)
             )) for u in user_set],

```

Comment:
Can we revise to use P_L_tot_us?
It is advisable to check also in the repository where else the "load" is used and see if a revision is possible